### PR TITLE
Bookmark

### PR DESCRIPTION
**Pull request by jeetbhalani15**
## Bookmark Post
- Added Bookmark page
- Add & remove bookmark using api

### CODE REVIEW

**Review by Jay-Gadhiya — APPROVED** [review]

File: src/Components/SuggestionCard/SuggestionCard.jsx [hunk 1]
```diff
@@ -30,10 +31,28 @@ function SuggestionCard() {
     dispatch(logoutUser());
     navigate("/");
   };
-  console.log(user.username)
+  console.log(user.username);
```
Comment (SHIKHASINGH1506) [inline src/Components/SuggestionCard/SuggestionCard.jsx:34]: you can remove logs

File: src/features/Posts/postSlice.js [hunk 2]
```diff
@@ -164,6 +165,47 @@ export const fetchDisLikePost = createAsyncThunk("post/fetchDisLikePost", async
...
+
+  } catch (error) {
+      console.error(error);
+
+  }
+})
+export const removeBookmark = createAsyncThunk("post/removeBookmark", async ({token, postId}) => {
+  try {
+    console.log(token)
+    console.log(postId)
```
Comment (SHIKHASINGH1506) [inline src/features/Posts/postSlice.js:192]: from here as well

File: src/Components/Card/Card.jsx [hunk 3]
```diff
@@ -4,17 +4,20 @@ import { BsHeart } from "react-icons/bs";
 import { FiMessageCircle } from "react-icons/fi";
 import { FiSend } from "react-icons/fi";
 import { BsBookmark } from "react-icons/bs";
+import { BsBookmarkCheckFill } from "react-icons/bs";
 import { RiMoreFill } from "react-icons/ri";
 import { MdDeleteOutline } from "react-icons/md";
 import logo from "../../Assets/Images/logo.png";
 import { useAuth } from "../../features/Auth/authSlice";
```
Comment (SHIKHASINGH1506) [inline src/Components/Card/Card.jsx:11]: you can use absolute import in your app by adding a jsconfig.json file. Relative imports can grow lengthy. Just a suggestion.

**Review by SHIKHASINGH1506 — APPROVED** [review]
1. liked use of tailwind 
2. code is modular and readable
3. just remove logs and good to merge

**Pull request merged**